deeplabcut/compat.py

```diff
@@ -1219,23 +1242,27 @@ def extract_maps(
             Indices=Indices,
             modelprefix=modelprefix,
         )
+    elif engine == Engine.PYTORCH:
+        from deeplabcut.pose_estimation_pytorch import extract_maps
+        extract_maps(
```

Comment:
```suggestion
        return extract_maps(
```
Otherwise it ends in the `raise NotImplementedError(...)`